lib/routes/lnurl/withdraw/lnurl_withdraw_dialog.dart

```diff
@@ -148,12 +148,12 @@ class _LNURLWithdrawDialogState extends State<LNURLWithdrawDialog> with SingleTi
         amountSats: widget.amountSats,
         description: description,
       );
-      if (resp is sdk.LnUrlWithdrawSuccessData) {
-        final paymentHash = (resp as sdk.LnUrlWithdrawSuccessData).invoice.paymentHash;
+      if (resp is sdk.LnUrlWithdrawResult_Ok) {
+        final paymentHash = (resp.data as sdk.LnUrlWithdrawSuccessData).invoice.paymentHash;
```

Comment:
We don't need to cast the type here as it's inferred by the if condition.

```suggestion
        final paymentHash = resp.data.invoice.paymentHash;
```